app/views/layouts/provider.html.slim

```diff
@@ -10,6 +10,8 @@ html[lang="en" class="pf-m-redhat-font"]
     = render 'provider/theme'
     = render 'provider/analytics'
     = javascript_include_tag 'provider/layout/provider'
+    / Orden matters! Base must go before any other pack:
```

Comment:
```suggestion
    / Order matters! Base must go before any other pack:
```

Reply:
Probably due to rebasing @mayorova's comment still applies